sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/st_functions.scala

```diff
@@ -271,4 +271,7 @@ object st_functions extends DataFrameAPI {
 
   def ST_ZMin(geometry: Column): Column = wrapExpression[ST_ZMin](geometry)
   def ST_ZMin(geometry: String): Column = wrapExpression[ST_ZMin](geometry)
+
+  def ST_S2CellIDs(geometry: Column, level: Int): Column = wrapExpression[ST_S2CellIDs](geometry, level)
+  def ST_S2CellIDs(geometry: String, level: Int): Column = wrapExpression[ST_S2CellIDs](geometry, level)
```

Comment:
(1) Please add simple tests for this DataFrame style API.
(2) Please add this DataFrame style API to Sedona Python as well.

Reply:
done, please check